code/game/machinery/cryopod.dm

```diff
@@ -280,9 +286,12 @@
 // Also make sure there is a valid control computer
 /obj/machinery/cryopod/proc/despawn_occupant()
 	var/mob/living/carbon/human/H = occupant
-	var/list/occupant_organs = H.organs | H.internal_organs
+	var/list/occupant_organs
```

Comment:
Does `W in occupant_organs` work correctly if `occupant_organs` is null?

Reply:
The for() will exit early when occupant_organs is null http://www.byond.com/docs/ref/index.html#/proc/for/list